.github/workflows/system-test-notification.yml

```diff
@@ -0,0 +1,32 @@
+name: system-test-notification
+
+on:
+  # This needs to be `pull_request_target` instead of `pull_request`
+  # to be able to write a comment on the pull request.
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  notify:
+    # This job runs only if the pull request is from a forked repository.
+    if: github.event.pull_request.head.repo.full_name != github.repository
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add comment
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const { repo, owner } = context.repo
+            await github.rest.issues.createComment({
```

Comment:
@axw 

Unfortunately, it was not able to use a status check, because the token does not have access to create a commit status on a forked PR.

This is why I just create a comment instead.

Reply:
No problem, thanks for explaining